cmd/pinsharesvc/process.go

```diff
@@ -0,0 +1,386 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sync"
+	"syscall"
+	"time"
+
+	"golang.org/x/sys/windows/svc/debug"
+)
+
+type ProcessManager struct {
+	config          *ServiceConfig
+	eventLog        debug.Log
+	ipfsCmd         *exec.Cmd
+	pinshareCmd     *exec.Cmd
+	ipfsLogFile     *os.File
+	pinshareLogFile *os.File
+	ipfsExited      chan struct{} // closed when IPFS process exits
+	pinshareExited  chan struct{} // closed when PinShare process exits
+	mu              sync.Mutex
```

Comment:
Let's rename this mutex to reflect the variables which it protects, and move it above their declarations.